Anamnesis/Files/PoseFile.cs

```diff
@@ -253,6 +253,20 @@ public async Task Apply(ActorMemory actor, SkeletonVisual3d skeleton, HashSet<st
 		PoseService.Instance.CanEdit = true;
 	}
 
+	public bool IsAPreDTPoseFile()
```

Comment:
this may be personal preference, but ive never seen "A" used in a method name. 
Id prefer this to just be "IsPreDTPoseFile" here.

Reply:
Changed the method name.